frontend/src/data_context.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 
 import {get} from './communication'
 
-import {DataContext, Genebank, NameID} from "./data_context_global"
+import {Colour, DataContext, Genebank, NameID} from "@app/data_context_global"
```

Comment:
Uh oh, this file uses both `Colour` and `Colors`. I would go for one of the spellings, AE in particular.

Reply:
yeah... it's mostly a nod to Andreas always using "colour" :) And the british spelling never collide with any other definitions ^^
I should probably change it to `Color` again though...